theme.conf

```diff
@@ -894,7 +894,7 @@
 24/apps chrome.svg
 24/apps chromium.svg
 24/apps book_dictionary.svg accessories-dictionary.svg
-24/apps edit.svg accessories-text-editor.svg leafpad.svg
+24/apps edit.svg accessories-text-editor.svg leafpad.svg16/actions edit.svg document-edit.svg
```

Comment:
mistake here

Reply:
Was an earlier mistake. Check now, it has been fixed.

Reply:
Github (or I) was too slow!

Reply:
None were slow, I suppose. Github doesn't reload changes in files on its own if files tab is already open, I think.